tket/src/MeasurementSetup/include/MeasurementSetup/MeasurementSetup.hpp

```diff
@@ -37,6 +38,7 @@ namespace tket {
 class MeasurementSetup {
  public:
   struct MeasurementBitMap {
+    MeasurementBitMap(){};
```

Comment:
Can we add a test for serialization/deserialization of a default-constructed `MeasurementBitMap`?